Update tutorial-classifying-iris-part-2.md

jainr opened the pull request:
az set account -s "test"
az: error: argument _command_package: invalid choice: set

Results in error, correct usage is az account set -s "test "

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

jainr committed:
Update tutorial-classifying-iris-part-2.md

az set account -s "test"
az: error: argument _command_package: invalid choice: set

Results in error, correct usage is az account set -s "test "
